pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
 ]
```

Comment:
Let's bump the version of pyadditive. It doesn't really matter but old habits die hard.

Reply:
Bumped.